app/services/gias_csv_importer.rb

```diff
@@ -145,30 +73,21 @@ def call
   def associate_schools_to_regions
     Rails.logger.debug "Associating schools to regions... "
 
-    Region.find_or_create_by!(name: "Inner London") { |region| region.claims_funding_available_per_hour = 53.60 }
-    Region.find_or_create_by!(name: "Outer London") { |region| region.claims_funding_available_per_hour = 48.25 }
-    Region.find_or_create_by!(name: "Fringe") { |region| region.claims_funding_available_per_hour = 45.10 }
-    Region.find_or_create_by!(name: "Rest of England") { |region| region.claims_funding_available_per_hour = 43.18 }
+    # Inner London Schools
+    inner_london = Region.find_or_create_by!(name: "Inner London") { |region| region.claims_funding_available_per_hour = 53.60 }
+    School.where(district_admin_code: INNER_LONDON_AREAS).update_all(region_id: inner_london.id)
```

Comment:
Presumably this works too? Again, not a strong opinion here, but it might be more idiomatic.

```suggestion
    School.where(district_admin_code: INNER_LONDON_AREAS).update_all(region: inner_london)
```

Reply:
I tried that and got 
```
TypeError: can't cast Region (TypeError)
```